packages/react/src/components/Checkbox/Checkbox.props.ts

```diff
@@ -0,0 +1,44 @@
+export interface CheckboxProps {
+
+  /**
+   * If `true`, the component is checked
+   * @default false
+   */
+  checked?: boolean
+
+  /**
+   * If `true`, the Checkbox will be disabled.
+   * @default false
+   */
+  disabled?: boolean
+
+  /**
+   * The id of the Checkbox element
+   * Must be the same as label
+   */
+  id: string
+
+  /**
+   * If `true`, indicates that the value of the checkbox is indeterminate
+   * rather than true or false
+   * @default false
+   */
+  indeterminate?: boolean
+
+  /**
+  * The onChange event occurs when the element is checked.
```

Comment:
checked e unchecked também né?

Reply:
É uma propriedade com valor `boolean`, não vejo sentido de duplicar.

Reply:
eu acho que faz sentido. onChange é sobre mudança de estado. o evento é chamado no momento que o elemento é "ligado" e também quando ele é "desligado".

Reply:
mas daí eu vou ficar com duas propriedades booleanas (checked/unchecked), onde uma é o oposto da outra?

Reply:
não! é só sobre o comentário! 😅

"The onChange event occurs when the element is checked **or unchecked**"

ou como tá no w3schools:
"The onchange event occurs when the checked state has been changed."
https://www.w3schools.com/jsref/event_onchange.asp